container/pureos/pkg.sh.cuda

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+set -e
+apt-get -y install curl wget git \
+	capnproto \
+	pkg-config \
+	libopenblas-{dev,base} \
+	cuda-cublas-dev-8-0 \
+	cuda-minimal-build-8-0 \
+	libcuda1-375 \
+	nvidia-375-dev
+
+dpkg -i /tmp/libcudnn7*.deb
+
```

Comment:
Do you really want the cudnn from the nvidia website which this will install in the future or does pureos provide `cudnn` in it's own repositories?

Reply:
pureos definitely doesn't provide it.